pyproject.toml

```diff
@@ -51,6 +51,15 @@ dynamic = [
   "version"
 ]
 
+[project.optional-dependencies]
+testing = [
+  "pytest",
+  "virtualenv",
+  "pytest-cov",
+  "pytest-qt",
+  "pytest-xvfb ; sys_platform == 'linux'"
```

Comment:
```suggestion
```

We use action for this (if it is present anywhere in other napari projects, please points). 

In general, in the past, we have had strange and hard to debug bugs connected with usage of `pytest-xvfb` and decided to run xvfb outside the python process.